### PR TITLE
Fix fossology workflow

### PR DESCRIPTION
**Pull request by tdrozdovsky**
# Description

Fix fossology error:  
```
Traceback (most recent call last):
  File "/bin/fossologyscanner", line 273, in <module>
    sys.exit(main(args))
  File "/bin/fossologyscanner", line 231, in main
    api_config = get_api_config()
  File "/bin/fossologyscanner", line 65, in get_api_config
    api_config.project_orig = os.environ['GITHUB_REPO_OWNER']
  File "/usr/lib/python3.9/os.py", line 679, in __getitem__
    raise KeyError(key) from None
KeyError: 'GITHUB_REPO_OWNER'
Error: Process completed with exit code 1.
```
## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] CI system update

**Test Configuration**:
* OS type & version: Ubuntu 20.04
* Hardware: x86-64 (e.g., x86, x86-64, arm, arm64)
* Toolchain: Docker v20.10 & Go v1.19
* Edge Orchestration Release: v1.2.x

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by tdrozdovsky** [comment]
I suggest looking at the error below in test-suite separately, since locally the tests work correctly, and catching this error will be quite difficult since it depends on the hardware where the github->action is running
```
ERRO[2023-12-12T16:40:55Z]cpu.go:63 readCPUUsage                                              
ERRO[2023-12-12T16:40:55Z]cpu.go:63 readCPUUsage                                              
ERRO[2023-12-12T16:40:55Z]cpu.go:189 getCPUFreqCPUInfo EOF                                          
ERRO[2023-12-12T16:40:55Z]cpu.go:189 getCPUFreqCPUInfo EOF                                          
ERRO[2023-12-12T16:40:55Z]cpu.go:189 getCPUFreqCPUInfo EOF                                          
--- FAIL: TestInfo (0.04s)
    --- FAIL: TestInfo/Success (0.04s)
        cpu_test.go:389: unexpected fail
FAIL
coverage: 96.3% of statements
FAIL	github.com/lf-edge/edge-home-orchestration-go/internal/common/resourceutil/cpu	1.045s
```

**Review by MoonkiHong — APPROVED** [review]
+1

**Comment by tiokim** [comment]
> I suggest looking at the error in test-suite separately, since locally the tests work correctly, and catching this error will be quite difficult since it depends on the hardware where the github->action is running
> 
> ```
> ERRO[2023-12-12T16:40:55Z]cpu.go:63 readCPUUsage                                              
> ERRO[2023-12-12T16:40:55Z]cpu.go:63 readCPUUsage                                              
> ERRO[2023-12-12T16:40:55Z]cpu.go:189 getCPUFreqCPUInfo EOF                                          
> ERRO[2023-12-12T16:40:55Z]cpu.go:189 getCPUFreqCPUInfo EOF                                          
> ERRO[2023-12-12T16:40:55Z]cpu.go:189 getCPUFreqCPUInfo EOF                                          
> --- FAIL: TestInfo (0.04s)
>     --- FAIL: TestInfo/Success (0.04s)
>         cpu_test.go:389: unexpected fail
> FAIL
> coverage: 96.3% of statements
> FAIL	github.com/lf-edge/edge-home-orchestration-go/internal/common/resourceutil/cpu	1.045s
> ```

What about excluding `resourceutil/cpu` test from the workflow?

**Comment by tdrozdovsky** [comment]
> > I suggest looking at the error in test-suite separately, since locally the tests work correctly, and catching this error will be quite difficult since it depends on the hardware where the github->action is running
> > ```
> > ERRO[2023-12-12T16:40:55Z]cpu.go:63 readCPUUsage                                              
> > ERRO[2023-12-12T16:40:55Z]cpu.go:63 readCPUUsage                                              
> > ERRO[2023-12-12T16:40:55Z]cpu.go:189 getCPUFreqCPUInfo EOF                                          
> > ERRO[2023-12-12T16:40:55Z]cpu.go:189 getCPUFreqCPUInfo EOF                                          
> > ERRO[2023-12-12T16:40:55Z]cpu.go:189 getCPUFreqCPUInfo EOF                                          
> > --- FAIL: TestInfo (0.04s)
> >     --- FAIL: TestInfo/Success (0.04s)
> >         cpu_test.go:389: unexpected fail
> > FAIL
> > coverage: 96.3% of statements
> > FAIL	github.com/lf-edge/edge-home-orchestration-go/internal/common/resourceutil/cpu	1.045s
> > ```
> 
> What about excluding `resourceutil/cpu` test from the workflow?

I'll try to fix this later. If it doesn’t work out then we’ll exclude it.

File: .github/workflows/fossology-check.yml [hunk 1]
```diff
@@ -1,33 +1,58 @@
 name: Fossology check
 on: [push, pull_request]
```
Comment (tiokim) [inline .github/workflows/fossology-check.yml:2]: What about running this on PR only?

**Review by tiokim — APPROVED** [review]
LGTM Thanks!

File: .github/workflows/fossology-check.yml (hunk 1 above)
Comment (tdrozdovsky) [reply .github/workflows/fossology-check.yml]: > What about running this on PR only?

Done!

**Pull request merged**